src/libraries/System.Linq.Queryable/tests/Queryable.cs

```diff
@@ -96,6 +96,7 @@ public static void MatchSequencePattern()
                      nameof(Enumerable.Prepend),
                      nameof(Enumerable.ToHashSet),
                      nameof(Enumerable.TryGetNonEnumeratedCount),
+                     nameof(Enumerable.Reverse),
```

Comment:
I wish there was a simple way to distinguish between the different overloads, but I suppose this should suffice.